data.py

```diff
@@ -0,0 +1,19 @@
+from csv import reader as csv_reader
```

Comment:
I would say let local variable change, the problem at here is others programmer might copy this line for convenience and make this case specific import statement became the shit in the code base, avoiding from in python